libc/test/src/sys/mman/linux/mincore_test.cpp

```diff
@@ -102,14 +106,14 @@ TEST(LlvmLibcMincoreTest, PageOut) {
   // page out the memory
   {
     libc_errno = 0;
-    EXPECT_THAT(LIBC_NAMESPACE::madvise(addr, EXEC_PAGESIZE, MADV_DONTNEED),
+    EXPECT_THAT(LIBC_NAMESPACE::madvise(addr, page_size, MADV_DONTNEED),
                 Succeeds());
 
     libc_errno = 0;
-    int res = LIBC_NAMESPACE::mincore(addr, EXEC_PAGESIZE, &vec);
+    int res = LIBC_NAMESPACE::mincore(addr, page_size, &vec);
```

Comment:
Your suggestion of changing this to:
```
int res = LIBC_NAMESPACE::mincore(addr, 1, &vec);
```
does fix the segfault in AArch64.  Otherwise, it is consistently crashed on AArch64.